base_layer/core/src/transactions/transaction.rs

```diff
@@ -108,9 +111,21 @@ pub struct OutputFeatures {
 }
 
 impl OutputFeatures {
-    pub fn to_bytes(&self) -> Vec<u8> {
-        let mut buf = Vec::new();
-        bincode::serialize_into(&mut buf, self).unwrap(); // this should not fail
+    /// Encodes output features for a v1 block
+    pub fn to_v1_bytes(&self) -> Vec<u8> {
```

Comment:
The default impl for  `OutputFeatures` no longer makes sense as the version depends on your network -> block version, which entails lots of changes. 

The intension was to remove the v1/v2 concept and just use consensus encoding at a convenient point (like a testnet reset).  But it may. be worth putting in the time to include "transaction/UTXO versioning" while maintaining backward-compatibility. Perhaps for another PR?